data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -287,12 +287,12 @@
             [/else]
         [/if]
 
-        [message]
+        [hint_message]
             speaker=narrator
             caption= _ "Recalling"
             image=wesnoth-icon.png
```

Comment:
Why doesn't the schema validation trigger for the leftover `speaker` and `image` here?

Reply:
The `[event]` tag is set to allow any child tag inside it. The effect is that the schema validates known ActionWML tags and ignores unknown ones. The reason it's defined this way is so that adding a custom tag doesn't automatically break the schema.

Reply:
Still, they're leftovers that you should fix.